point_pattern.py

```diff
@@ -95,6 +103,12 @@ def mean_center(points):
     """
     x = None
     y = None
+    for point in points:
+        x += [0]
+        y += [1]
```

Comment:
These two lines say, add x (or y) to my list.  I think what you want to do is add x (or y) to the appropriate point coordinate, e.g. point[0], or point[1].